packages/frontend/src/components/Accordion/config.ts

```diff
@@ -0,0 +1,22 @@
+import { recipeType } from "src/types/recipeType.types";
+
+export const recipeTypesArr: recipeType[] = [
```

Comment:
Te tagi powinny być pobierane z api. W api nie masz pola isOpen, więc musisz to inaczej wymyślić

Reply:
zgadzam się

Reply:
taki nię będą się zmieniać więc cache też powinien być dłuższy ustawiony jak coś